react/src/js/components/Consonant/Cards/Card.jsx

```diff
@@ -210,6 +210,9 @@ const Card = (props) => {
     } else if (detailsTextOption === 'createdDate' && cardDate) {
         const localCreatedDate = new Date(cardDate);
         detailText = localCreatedDate.toLocaleDateString();
+    } else if (detailsTextOption === 'staticDate' && cardDate) {
+        const staticDate = new Date(cardDate.replace(/Z$/, ''));

```

Comment:
This is a very clean solution!

Much better than just splitting by `T` and taking the left hand (as it'll also localize the dates in the right format)